Week2/connection_query.js

```diff
@@ -0,0 +1,29 @@
+//C:\Users\knowl\Documents\hyf\databases\Week2\connection_query.js
+import mysql from 'mysql';
+
+export const createNewConnection = () => {
```

Comment:
I like how you are thinking here, but please keep in mind everytime you call this function gives an opportunity to create a new connection. Meaning you can easily overwhelm the MySQL server in a real use case scenario. You can allow only one object of connection to be passed around, this is called [Singleton pattern](https://en.wikipedia.org/wiki/Singleton_pattern) or a fixed number of connection objects to be passed around, this is called [Connection pooling](https://en.wikipedia.org/wiki/Connection_pool). 

But for the purposes of this exercise its okay 😄